toc/rfc/rfc-0007-repository-ownership.md

```diff
@@ -55,24 +55,94 @@ The TOC MUST also approve of any proposal to create a new repository within the
-    * The TOC MUST approve the addition of the proposed repository to the Working Group's subset.
-
-1. The TOC wants Working Group A to take ownership of an unclaimed active repository.
-    * The TOC MUST approve the change in the ownership scope of Working Group A.
-    * Working Group A MUST approve the change to the set of repositories they own.
-
-1. Working Group A wants to claim ownership of a repository currently owned by Working Group B.
-    * Both Working Group A and Working Group B MUST approve the change to the sets of repositories they own.
-    * The TOC MUST also approve the transfer of the repository from Working Group B to Working Group A.
-
-1. A Working Group wants to cease maintaining a repository that it currently owns.
-    * The Working Group MUST approve the removal of the repository from the set of repositories that it owns.
-    * The TOC MUST approve the removal of the repository from the Working Group.
-    * In addition, the TOC MUST decide whether to archive the repository or to arrange for another Working Group to own it instead (possibly a newly formed one, if appropriate and if supported by community contributors).
-    * **Note:** In this case, it is unlikely that either the TOC or the CFF staff should own this repository, as it was already owned by some Working Group in the CF community.
-
-1. A Working Group wants to rename a repository that it currently owns.
-    * The Working Group A and the TOC MUST both approve of the new name for the repository.
+#### Case 1: A Working Group wants to create a new repository under their ownership.
+
+Approval requirements
+* The Working Group MUST approve the addition of the proposed repository to the set of repositories that it owns.
+* The TOC MUST approve the creation of the proposed repository.
+* The TOC MUST approve the addition of the proposed repository to the Working Group's subset.
+
+Steps
+1. Someone opens two PRs. The first PR adds the repo to the [cloudfoundry.yml](../../org/cloudfoundry.yml). The second PR adds the repo to the desired Working Group's yaml. These PRs should link to each other.
+2. The Working Group and TOC both approve both PRs.
+3. The TOC merges the first PR.
+4. Wait for automation to run to create the repo.
+5. The TOC merges the second PR.
+
+#### Case 2: The TOC wants Working Group A to take ownership of an unclaimed active repository.
+
+Approval requirements
+* The TOC MUST approve the change in the ownership scope of Working Group A.
+* Working Group A MUST approve the change to the set of repositories they own.
+
+Steps
+1. Someone opens a PRs that adds a repo to the desired Working Group's yaml.
+2. The Working Group and TOC both approve the PR.
+3. The TOC merges the PR.
+
+
+#### Case 3: Working Group A wants to claim ownership of a repository currently owned by Working Group B.
+
+Approval requirements
+* Both Working Group A and Working Group B MUST approve the change to the sets of repositories they own.
```

Comment:
It would be great to call out which role and if it needs to be unanimous or if one vote per working group would be sufficient.

Reply:
This is defined above:

> Approval consists of a quorum decision by the leads of the Working Group.